arbitrator/wasm-libraries/user-test/src/host.rs

```diff
@@ -25,6 +25,7 @@ pub unsafe extern "C" fn vm_hooks__read_args(ptr: u32) {
 pub unsafe extern "C" fn vm_hooks__write_result(ptr: u32, len: u32) {
     let mut program = Program::start(0);
     program.pay_for_read(len).unwrap();
+    program.pay_for_geth_bytes(len).unwrap();
```

Comment:
Paying for the copy on the Go side below, right?

Reply:
Yeah, that's what that's for, though this test version of `user-host` doesn't actually have a Go side